aws_google_auth/google.py

```diff
@@ -6,9 +6,12 @@
 import sys
 import requests
 import json
+import io
 import base64
 from bs4 import BeautifulSoup
+from PIL import Image
 from six.moves import urllib_parse, input
+from six.moves.urllib.request import urlopen
```

Comment:
We're already importing `requests` for HTTP requests, could we refactor to not need the `urlopen`? Just a thought on reducing dependencies.

Reply:
https://github.com/louahola/aws-google-auth/pull/3/files#diff-ad879f154c6e5e24eb1f5e7615f90866R316